ci/do_ci.sh

```diff
@@ -92,7 +92,7 @@ if [[ $# -gt 1 ]]; then
   shift
   TEST_TARGETS=$*
 else
-  TEST_TARGETS=//test/...
+  TEST_TARGETS="//test/... $(bazel query ${BAZEL_QUERY_OPTIONS} "attr('size', 'small', '@com_googlesource_quiche//:all')" | grep "^@com_googlesource_quiche")"
```

Comment:
I changed to check a new tag 'test_included'. We want to run QUICHE tests selectively because we may add a test target quic_end_to_end_test which is can be ran manually in case of need.